tests/e2e/support/objects/app-files/link/index.ts

```diff
@@ -84,9 +84,6 @@ export class Link {
   async delete(args: Omit<deleteLinkArgs, 'page'>): Promise<void> {
     const startUrl = this.#page.url()
     await deleteLink({ ...args, page: this.#page })
-    this.#linksEnvironment.deleteLink({
-      key: args.name
-    })
```

Comment:
We still need to delete it from the internal store, don't we?

Reply:
no, not nessasary. after test we have step `createdLinkStore.clear()`